packages/noya-state/src/selectors/__tests__/gradientSelectors.test.ts

```diff
@@ -0,0 +1,217 @@
+import Sketch from '@sketch-hq/sketch-file-format-ts';
+import { SketchModel } from 'noya-sketch-model';
+import { createInitialState } from '../../reducers/applicationReducer';
+import { createSketchFile } from '../../sketchFile';
+import {
+  getAngularGradientCircle,
+  getSelectedGradientStopPoints,
+} from '../gradientSelectors';
+
+const angularGradientStyle = SketchModel.style({
+  fills: [
+    SketchModel.fill({
+      fillType: Sketch.FillType.Gradient,
+      gradient: SketchModel.gradient({
+        gradientType: Sketch.GradientType.Angular,
+      }),
+    }),
+  ],
+});
+
+const linearGradienStyle = SketchModel.style({
+  fills: [
+    SketchModel.fill({
+      fillType: Sketch.FillType.Gradient,
+      gradient: SketchModel.gradient(),
+    }),
+  ],
+});
+
+// Gradient Selector
+describe('set angular gradient circle', () => {
+  const rectangle = SketchModel.rectangle({
+    frame: SketchModel.rect({
+      x: 0,
+      y: 0,
+      width: 100,
+      height: 100,
+    }),
+    style: angularGradientStyle,
+  });
+
+  test('top layer', () => {
+    const state = createInitialState(
+      createSketchFile(SketchModel.page({ layers: [rectangle] })),
+    );
+    state.selectedGradient = {
+      layerId: rectangle.do_objectID,
+      fillIndex: 0,
+      stopIndex: 0,
+      styleType: 'fills',
+    };
+
+    expect(getAngularGradientCircle(state)).toEqual({
+      center: { x: 50, y: 50 },
+      radius: 50,
+      rotation: 0,
+    });
+  });
+  test('fliped layer', () => {
```

Comment:
flipped

Reply:
We should also test `rotation`, which is different than flipping. And maybe `isFlippedVertical` too?